dev-tools/commands/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:14
+
+WORKDIR /usr/src/app
```

Comment:
Using [this pattern](https://github.com/AudiusProject/audius-protocol/blob/master/monitoring/exporters/alchemy/Dockerfile#L3-L4) is the recommended community approach since it allows us to rewrite lines 6 and 7 using `${WORKDIR}` instead of hardcoding `/usr/src/app`.